src/core/geometry/qgslinestring.cpp

```diff
@@ -711,6 +711,158 @@ QgsLineString *QgsLineString::reversed() const
   return copy;
 }
 
+QgsPoint *QgsLineString::interpolatePoint( const double distance ) const
```

Comment:
> const double distance

Nice :). Is the *const correctness* a goal? I mean, do we want to check it during reviews?

Reply:
I think it's good practice for new code, but there's SO many existing instances where const is missing that I don't think it's worth including in reviews.

What we really need is for clang-tidy or clazy to implement an automatic fixit for this situation so that we can auto add const to all existing code (when it's valid), and only then move to using const as common practice.